downloads/github-actions-cheat-sheet.md

```diff
@@ -0,0 +1,193 @@
+---
+layout: cheat-sheet
+redirect_to: false
+title: GitHub Actions Cheat Sheet
+byline: GitHub Actions give you the flexibility to build automated software development lifecycle workflows. You can write individual tasks, called actions, and combine them to create  custom workflows in your repository. GitHub Actions are automated processes allowing you to build, test, package, release, or deploy any code project on GitHub, but you can also use them to automate any step of your workflow&#58; merging pull requests, assigning labels, triaging issues to name a few.
+leadingpath: ../../
+---
+
+{% capture colOne %}
+## Workflow Syntax
+Workflow files use YAML syntax, and must have either a .yml or .yaml file extension. You must store workflow files in the `.github/workflows/` directory of your repository.
+
+```yaml
+name: Node CI
+on: push
+  branches:    
+    - 'releases/*'
+jobs:
+  my_build:
+    runs-on: ubuntu-latest
+    needs: my-other-job
+    steps:
+    - uses: actions/checkout@master
+    - name: Say something
+      run: echo "A little bit more action"
+```
+
+### Workflow `name`
+The name of your workflow. GitHub displays the names of your workflows on your repository's actions page.
```

Comment:
Are there any naming rules? Forbidden characters?  Nothing in the docs but I wonder if special chars are forbidden.